src/briefcase/integrations/xcode.py

```diff
@@ -469,12 +471,7 @@ def get_identities(command, policy):
             ['security', 'find-identity', '-v', '-p', policy],
         )
 
-        identities = dict(
-            IDENTITY_RE.match(line).groups()
-            for line in output.split('\n')
-            if IDENTITY_RE.match(line)
-        )
+        return dict(IDENTITY_RE.match(line).groups() for line in output.split('\n') if IDENTITY_RE.match(line))
```

Comment:
Readability (plus... is there any reason to not do this a dict comprehension with `{}`?)